libs/frontend/feature-build/src/lib/use-toolbar.ts

```diff
@@ -74,15 +83,23 @@ export const useToolbar = (): IUseToolbar => {
 
   const createSetComponentCustomStyleCommand = (
     property: Css,
-    oldValue: string | undefined,
     newValue: string | undefined,
-  ): ICommand<ISetComponentCustomStyleData> => {
+  ): ICommand<ISetComponentCustomStyleData> | undefined => {
     const { selectedComponent } = editor.value ?? {}
     if (!selectedComponent) {
       throw new Error(
         'Cannot create set component custom style command when no component is selected',
       )
     }
+    const oldStyle = selectedComponentFlattenedStyles.value[property]
```

Comment:
Previously, `createSetComponentCustomStyleCommand` accepted the oldValue, and we were passing in the old style after resolving theme variables. This caused a problem with undo, since changing a themed color, then undoing, would result in the resolved color instead of the variables.